katdal/visdatav4.py

```diff
@@ -79,21 +79,22 @@ def _calc_azel(cache, name, ant):
 def _calc_delay(cache, name, inp):
     """Extract virtual applied delay/phase sensors from raw CBF sensors."""
     # Obtain the relevant CBF attributes from the cache
-    instrument = cache.get('Correlator/instrument')[0]
+    stream = cache.get('Correlator/f_engine_stream')[0]
```

Comment:
Officially this is called "antenna channelised voltage" - not just in documentation, but also in CAM sensors, SDP-CAM ICD (for stream types), and so on. It's probably best to be consistent.

Reply:
I see your point. I've slipped up once already in katcorelib...